### PR TITLE
Feature/46 idempotency keys

### PR DESCRIPTION
**Pull request by taryneast**
API-doc explaining the new idempotency keys changes for our customers.

See: https://trello.com/c/1eEUYzXX

Do NOT merge until the Sendle epic/46-idempotency_keys epic is merged!

### CODE REVIEW

**Comment by taryneast** [comment]
[API idempotency keys](https://trello.com/c/1eEUYzXX/46-api-idempotency-keys)

**Comment by taryneast** [comment]
[API idempotency keys](https://trello.com/c/1eEUYzXX/46-api-idempotency-keys)

File: source/includes/_ping_server.md [hunk 1]
```diff
@@ -3,6 +3,37 @@
 
 > [ GET /api/ping ]
```
Comment (jstirk) [inline source/includes/_ping_server.md]: Are we now expecting this to be a POST?

Comment (taryneast) [reply source/includes/_ping_server.md]: nah, at present it's still just get. I reckon we could make it an "any", though.

**Comment by taryneast** [comment]
yep can do re: flag

**Pull request merged**